docs/GUIDE.md

```diff
@@ -0,0 +1,379 @@
+# Add Realtime 3D Avatars to Live Video Streams
+
+In today's rapidly evolving digital landscape, live-stream video is dominating real-time communication. Users now expect more immersive and customizable streaming options. Content creators are increasingly seeking creative new ways to stream themselves, giving rise to the demand for dynamic 3D avatars that mirror their movements and expressions.
+
+![](/docs/images/Intro-Demo.gif "Real-Time Avatars with Agora")
+
+Real-time virtual avatars traditionally required complex motion capture equipment and sophisticated software, often making them inaccessible to everyday users and independent creators. However, artificial intelligence has changed this status quo as well. With advancements in computer vision, it's now possible to run sophisticated AI algorithms on-device that can accurately capture and translate human facial gestures into digital form in real-time.
```

Comment:
However, artificial intelligence has changed this status quo ~as well~.

Removing this makes it sound more confident and concise

Reply:
updated to:

 However, this is another area where artificial intelligence is changing the status quo.